client/src/document/types.tsx

```diff
@@ -18,11 +18,25 @@ export interface BadBCDLink {
   key: string;
 }
 
+export interface GenericFlaw {
```

Comment:
The idea is that `LinkFlaw` can extend this too so it can use the common hook to create those annotations. 
Later.